src/docker-in-docker/devcontainer-feature.json

```diff
@@ -50,6 +45,16 @@
             "type": "boolean",
             "default": true,
             "description": "Install Docker Buildx"
+        },
+        "mobyBuildxVersion": {
```

Comment:
Can we add test scenarios to validate/massage these two options? thanks!

Reply:
Sure, will do